comet-examples/src/main/java/ml/comet/examples/RegistryModelExample.java

```diff
@@ -185,6 +185,14 @@ private static void run(OnlineExperiment experiment) throws Exception {
             Optional<String> notes = api.getRegistryModelNotes(registryName, experiment.getWorkspaceName());
             notes.ifPresent(s -> System.out.printf("Retrieved model notes: '%s'\n", s));
 
+            // update registry model
+            //
+            System.out.printf("Updating registry model '%s/%s'.\n", experiment.getWorkspaceName(), SOME_MODEL_NAME);
+            String newModelName = "updatedModelNameExample";
+            String newDescription = SOME_MODEL_DESCRIPTION + " updated";
+            api.updateRegistryModel(registryName, experiment.getWorkspaceName(), newModelName, newDescription);
```

Comment:
This only work once, the second time the registry model name `updatedModelNameExample` already exists, could we add a random value to the `newModelName`?